doc/docs/master/pachhub/pachhub_getting_started.md

```diff
@@ -1,181 +1,101 @@
-
-To log in to Pachyderm Hub, complete the following steps:
-
-1. Go to [hub.pachyderm.com](https://hub.pachyderm.com).
-1. Click **Try for free**.
-1. Authorize Pachyderm Hub with your GitHub account by typing your
-   GitHub user name and password.
-1. Proceed to [Step 1](#step-1-create-a-cluster).
-
-## Step 1: Create a Cluster
-
-To get started, create a Pachyderm cluster on which your pipelines will run.
-A Pachyderm cluster runs on top of the underlying cloud infrastructure.
-In Pachyderm Hub, you can create a one-node cluster that you can use for
-a limited time.
-
-To create a Pachyderm cluster, complete the following steps:
-
-1. If you have not yet done so, log in to Pachyderm Hub.
-1. Click **Create cluster**.
-1. Type a name for your cluster. For example, `test1`.
-1. Click **Create**.
-
-   Your cluster is provisioned instantly!
-
-   ![PachHub cluster](../assets/images/s_pachhub_cluster.png)
-
-   **Note:** While Pachyderm maintains a few clusters that are instantly
-   available, none may be available during periods of high traffic. If
-   you see your cluster is in a *starting* state, you might have to wait a few
-   minutes for it to be ready.
-
-1. Proceed to [Step 2](#step-2-connect-to-your-cluster).
-
-## Step 2 - Connect to Your Cluster
-
-Pachyderm Hub enables you to access your cluster through a command-line
-interface (CLI) called `pachctl` and the web interface called the Dashboard.
-Although you can perform most simple actions directly in the dashboard,
+and can get started right away.
+
+!!! Note
+    Let us know how to improve Pachyderm Hub. Join our [Slack channel](http://slack.pachyderm.io).
+## Get started in 4 simple steps
+![Pachyderm Hub Steps](../images/pachhub_steps.png)
+## Before you start
+Log in with your GitHub or Google account to start using [hub.pachyderm.com](https://hub.pachyderm.com). 
+![Pachyderm Hub Login](../images/pachhub_login.png)
+## 1- Create a Cluster 
+Click the **Create a 4-hr Workspace** button and fill out the form.
+![PachHub cluster](../images/pachhub_create_workspace.png)
+
+You just provisioned a one-node cluster that you can now use for
+a limited time for free!
+
+!!! Note
+      While Pachyderm maintains a few clusters that are instantly
+      available, none may be available during periods of high traffic. If
+      you see your cluster is in a *starting* state, you might have to wait a few
+      minutes for it to be ready.
+
+## 2- Install pachctl
+Pachyderm Hub enables you to access your cluster through Pachyderm 
+CLI `pachctl` and the web interface called the Dashboard.
+Although you can perform most simple actions directly in the Dashboard,
 `pachctl` provides full functionality. Most likely, you will use
 `pachctl` for any operation beyond the most basic workflow.
-Pachyderm recommends that you use `pachctl` for all data operations and
-the dashboard to view your data and graphical representation of your
+We recommend that you use `pachctl` for all data operations and
```

Comment:
I think there is some documentation style guide floating around that says to never use "we" in documentation... but I hate that convention and prefer what you wrote here ;)